embedded/src/main/java/org/wildfly/core/embedded/EmbeddedHostControllerFactory.java

```diff
@@ -271,10 +270,10 @@ public void exit(int status) {
                     HostControllerEnvironment environment = createHostControllerEnvironment(jbossHomeDir, cmdargs, startTime);
 
                     FutureServiceContainer futureContainer = new FutureServiceContainer();
-                    final byte[] authBytes = new byte[ProcessController.AUTH_BYTES_LENGTH];
```

Comment:
What's the reason for not continuing to use `ProcessController.AUTH_BYTES_LENGTH here`?

If we want as it is now, then the compile-time maven dependency on wildfly-process-controller could be also removed: https://github.com/wildfly/wildfly-core/blob/main/embedded/pom.xml#L198-L211